### PR TITLE
agenda pdf export

### PR DESCRIPTION
**Pull request by MaximilianKrambach**
Agenda pdf export, mostly from 2.3

not 100% sure the going through a tree is overkill now that we have the agenda hierarchy already in place, but I don't know a good way of telling 'is a parent hidden' there.

### CODE REVIEW

File: client/src/app/site/agenda/services/agenda-pdf.service.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                    {
+                        width: level * 15,
+                        text: ''
+                    },
+                    {
+                        width: 60,
+                        text: nodeItem.item.itemNumber
+                    },
+                    {
+                        text: nodeItem.item.getListTitle()
```
Comment (emanuelschuetze) [inline client/src/app/site/agenda/services/agenda-pdf.service.ts]: getListTitle contains also the itemNumber which is already in your first column!

Comment (emanuelschuetze) [reply client/src/app/site/agenda/services/agenda-pdf.service.ts]: Please use a bigger font size for items - like in 2.3 agenda pdf.

**Comment by MaximilianKrambach** [comment]
The font size could only be roughly adapted (different font), but I think it's better now @emanuelschuetze

**Comment by MaximilianKrambach** [comment]
rebased @tsiegleauq

**Pull request merged**